### ISSUE TITLE
[BUG] loss_obj在第一个step很大 然后变小 几个step后就nan

### ISSUE THREAD

**Issue by Fight-hawk** [PaddlePaddle/PaddleDetection#5167]
**PaddleDetection team appreciate any suggestion or problem you delivered~**

## Checklist:

1. 查找历史相关issue寻求解答/I have searched related issues but cannot get the expected help.
2. 翻阅[FAQ](https://paddledetection.readthedocs.io/FAQ.html) /I have read the [FAQ documentation](https://paddledetection.readthedocs.io/FAQ.html) but cannot get the expected help.
3. 确认bug是否在新版本里还未修复/The bug has not been fixed in the latest version.

## 描述问题/Describe the bug
kaggle比赛数据集，自己转成了coco格式。在其他框架下训练没问题，loss_obj在第一个step很大 然后变小 几个step后就nan
## 复现/Reproduction

1. 您使用的命令是？/What command or script did you run?

```none
 python -m paddle.distributed.launch --log_dir=./ppyolo_dygraph/ --gpus 4,5,6,7 tools/train.py -c configs/ppyolo/ppyolo_r50vd_dcn_1x_coco.yml.
```
2. 您是否更改过代码或配置文件？您是否理解您所更改的内容？还请您提供所更改的部分代码。 

```
metric: COCO
num_classes: 1

TrainDataset:
  !COCODataSet
    image_dir: train2017
    anno_path: annotations/train.json
    dataset_dir: /data_lvm/khp/data/coco
    data_fields: ['image', 'gt_bbox', 'gt_class']

EvalDataset:
  !COCODataSet
    image_dir: val2017
    anno_path: annotations/val.json
    dataset_dir: /data_lvm/khp/data/coco

TestDataset:
  !ImageFolder
    anno_path: annotations/val.json
```
输入size 是1920

3. 您使用的数据集是？/What dataset did you use?

4. 请提供您出现的报错信息及相关log。

```
[01/28 03:14:39] ppdet.engine INFO: Epoch: [0] [  0/276] learning_rate: 0.000000 loss_xy: 1.081778 loss_wh: 3.691975 loss_iou: 4.518171 loss_iou_aware: 0.929395 loss_obj: 150384.406250 loss_cls: 1.304925 loss: 150395.937500 eta: 4 days, 13:33:47 batch_cost: 3.5286 data_cost: 0.0005 ips: 0.5668 images/s
[01/28 03:15:11] ppdet.engine INFO: Epoch: [0] [ 20/276] learning_rate: 0.000250 loss_xy: 2.317560 loss_wh: 3.268679 loss_iou: 6.522950 loss_iou_aware: 1.817235 loss_obj: 397.890320 loss_cls: 1.332347 loss: 416.495758 eta: 2 days, 1:44:08 batch_cost: 1.5058 data_cost: 0.0003 ips: 1.3282 images/s
[01/28 03:15:45] ppdet.engine INFO: Epoch: [0] [ 40/276] learning_rate: 0.000500 loss_xy: nan loss_wh: nan loss_iou: nan loss_iou_aware: nan loss_obj: nan loss_cls: nan loss: nan eta: 2 days, 1:49:32 batch_cost: 1.6086 data_cost: 0.0013 ips: 1.2433 images/s
[01/28 03:16:22] ppdet.engine INFO: Epoch: [0] [ 60/276] learning_rate: 0.000750 loss_xy: nan loss_wh: nan loss_iou: nan loss_iou_aware: nan loss_obj: nan loss_cls: nan loss: nan eta: 2 days, 3:25:52 batch_cost: 1.7640 data_cost: 0.0003 ips: 1.1338 images/s
[01/28 03:16:58] ppdet.engine INFO: Epoch: [0] [ 80/276] learning_rate: 0.001000 loss_xy: nan loss_wh: nan loss_iou: nan loss_iou_aware: nan loss_obj: nan loss_cls: nan loss: nan eta: 2 days, 4:04:34 batch_cost: 1.7427 data_cost: 0.0016 ips: 1.1476 images/s
[01/28 03:17:36] ppdet.engine INFO: Epoch: [0] [100/276] learning_rate: 0.001250 loss_xy: nan loss_wh: nan loss_iou: nan loss_iou_aware: nan loss_obj: nan loss_cls: nan loss: nan eta: 2 days, 4:39:55 batch_cost: 1.7758 data_cost: 0.0003 ips: 1.1263 images/s
[01/28 03:18:12] ppdet.engine INFO: Epoch: [0] [120/276] learning_rate: 0.001500 loss_xy: nan loss_wh: nan loss_iou: nan loss_iou_aware: nan loss_obj: nan loss_cls: nan loss: nan eta: 2 days, 4:52:58 batch_cost: 1.7420 data_cost: 0.0012 ips: 1.1481 images/s
[01/28 03:18:49] ppdet.engine INFO: Epoch: [0] [140/276] learning_rate: 0.001750 loss_xy: nan loss_wh: nan loss_iou: nan loss_iou_aware: nan loss_obj: nan loss_cls: nan loss: nan eta: 2 days, 5:10:33 batch_cost: 1.7737 data_cost: 0.0003 ips: 1.1276 images/s
[01/28 03:19:27] ppdet.engine INFO: Epoch: [0] [160/276] learning_rate: 0.002000 loss_xy: nan loss_wh: nan loss_iou: nan loss_iou_aware: nan loss_obj: nan loss_cls: nan loss: nan eta: 2 days, 5:21:19 batch_cost: 1.7638 data_cost: 0.0011 ips: 1.1339 images/s
```

## 环境/Environment
1. 请提供您使用的Paddle和PaddleDetection的版本号 Paddle 2.2 PaddleDetection 2.3

2. 如您在使用PaddleDetection的同时还在使用其他产品，如PaddleServing、PaddleInference等，请您提供其版本号/ Please provide the version of any other related tools/products used, such as the version of PaddleServing and etc：

3. 请提供您使用的操作系统信息，docker镜像

4. 请问您使用的Python版本是？/ Please provide the version of Python you used.

5. 请问您使用的CUDA/cuDNN的版本号是？cuDNN Version: 7.6.

如果您的issue是关于安装或环境，您可以先查询[安装文档](https://github.com/PaddlePaddle/PaddleDetection/blob/release/2.1/docs/tutorials/INSTALL_cn.md)尝试解决~

If your issue looks like an installation issue / environment issue,
please first try to solve it yourself with the instructions in
https://github.com/PaddlePaddle/PaddleDetection/blob/release/2.1/docs/tutorials/INSTALL.md

**Comment by nemonameless** [comment]
训自定义数据集，推荐加载coco训好的模型权重作为预训练加速收敛。
看log是数据集太小，而且配置文件里的lr和和warmup默认是8卡训COCO设置的，需要调整比如lr减半 warmup step减少或换warmup策略。
